Commands/Branding/ApplyProvisioningTemplate.cs

```diff
@@ -60,127 +60,143 @@ public class ApplyProvisioningTemplate : SPOWebCmdlet
         protected override void ExecuteCmdlet()
         {
             SelectedWeb.EnsureProperty(w => w.Url);
-
-            if (!System.IO.Path.IsPathRooted(Path))
-            {
-                Path = System.IO.Path.Combine(SessionState.Path.CurrentFileSystemLocation.Path, Path);
-            }
-            if (!string.IsNullOrEmpty(ResourceFolder))
+            bool templateFromFileSystem = !Path.StartsWith("http");
```

Comment:
I guess there is a slight chance that someone writes HTTP. Should I fix and do a new PR?